From f5a346378152ec03ebeebc5731a041c264d20285 Mon Sep 17 00:00:00 2001
From: Sultanxda <sultanxda@gmail.com>
Date: Sun, 20 Dec 2015 13:16:21 -0800
Subject: [PATCH 072/206] SnapdragonCamera: Fix UI alignment glitches when
 nav-bar is enabled

Snapdragon Camera is immersive (i.e. fullscreen), so we expect to receive the
actual size of the display here, not the modified size when the nav-bar is
enabled.

Change-Id: I942829f77dda0dbb9dd65a60e66a540eecf708e7
Signed-off-by: Sultanxda <sultanxda@gmail.com>
---
 src/com/android/camera/PhotoUI.java | 2 +-
 src/com/android/camera/VideoUI.java | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/com/android/camera/PhotoUI.java b/src/com/android/camera/PhotoUI.java
index f6d302d55..4ce7a4f9b 100644
--- a/src/com/android/camera/PhotoUI.java
+++ b/src/com/android/camera/PhotoUI.java
@@ -309,7 +309,7 @@ public class PhotoUI implements PieListener,
         mPrevOrientationResize = false;
 
         Point size = new Point();
-        mActivity.getWindowManager().getDefaultDisplay().getSize(size);
+        mActivity.getWindowManager().getDefaultDisplay().getRealSize(size);
         mScreenRatio = CameraUtil.determineRatio(size.x, size.y);
         calculateMargins(size);
         mCameraControls.setMargins(mTopMargin, mBottomMargin);
diff --git a/src/com/android/camera/VideoUI.java b/src/com/android/camera/VideoUI.java
index b08e1db8e..6d326bb2f 100755
--- a/src/com/android/camera/VideoUI.java
+++ b/src/com/android/camera/VideoUI.java
@@ -293,7 +293,7 @@ public class VideoUI implements PieRenderer.PieListener,
         mPrevOrientationResize = false;
 
         Point size = new Point();
-        mActivity.getWindowManager().getDefaultDisplay().getSize(size);
+        mActivity.getWindowManager().getDefaultDisplay().getRealSize(size);
         mScreenRatio = CameraUtil.determineRatio(size.x, size.y);
         calculateMargins(size);
         mCameraControls.setMargins(mTopMargin, mBottomMargin);
-- 
2.17.1

